synnax/pkg/distribution/channel/create_test.go

```diff
@@ -143,4 +143,42 @@ var _ = Describe("Create", Ordered, func() {
 			Expect(ch.Key().Leaseholder()).To(Equal(aspen.Free))
 		})
 	})
+	Context("Updating a channel", func() {
```

Comment:
These cases don't actually test is retrieving the created channel actually has the updated expression.